src/entities/SecurityToken/Issuance/Offerings.ts

```diff
@@ -141,13 +144,17 @@ export class Offerings extends SubModule {
   /**
    * Launch a Tiered STO
    *
+   * @param args.tiers - array that specifies how many tokens to sell at each tier (along with their price and potential discounts)
    * @param args.nonAccreditedInvestmentLimit - maximum investment for non-accredited investors
    * @param args.minimumInvestment - minimum investment amount
    * @param args.currencies - array of currencies in which the funds will be raised (ETH, POLY, StableCoin)
    * @param args.raisedFundsWallet - wallet address that will receive the funds that are being raised
    * @param args.unsoldTokensWallet - wallet address that will receive unsold tokens when the end date is reached
    * @param args.stableCoinAddresses - array of stable coins that the offering supports
    * @param args.customCurrency - it can be optional
+   * @param customCurrency.currencySymbol symbol of the custom currency (USD, CAD, EUR, etc. Default is USD)
```

Comment:
You misunderstood. The only entry here should be for `args.customCurrency`. My comment was aimed at the fact that that entry had a long (and correct) description, and you just shortened it to "it can be optional"